data/drak/drak outfits.txt

```diff
@@ -134,7 +228,39 @@ outfit "Drak Antimatter Cannon"
 		"firing heat" 1870
 		"blast radius" 100
 		"hit force" 1000
-	description "This is the Drak's ultimate weapon."
+		safe
+	description "This is the most powerful weapon that the Archons deign to utilize in the presence of the younger races."
+
+
+
+outfit "Drak Antimatter Cannon - Augmented"
+	category "Guns"
+	cost 10000000
+	thumbnail "outfit/unknown"
+	"mass" 96
+	"outfit space" -96
+	"weapon capacity" -96
+	"gun ports" -1
+	"inscrutable" 1
+	"Drak Ammunition capacity" 10000000
+	weapon
+		ammo "Drak Ammunition"
+		"inaccuracy" 0
+		"velocity" 20
+		"lifetime" 1
+		"submunition" "Drak Antimatter Fragment - Augmented" 5
+		"hit effect" "antimatter dust" 10
+		"hit effect" "nuke explosion"
+		"safe"
+		"reload" 187
+		"firing energy" 5610
+		"firing heat" 1870
+		"blast radius" 250
+		"hit force" 1000
+		safe
```

Comment:
Another duplicate `safe`, missed it before.

Reply:
Thanks. And then I found a second one, too. I think all the places I added safes were actually duplicates.